test/WebSites/LowercaseUrlsWebSite/Startup.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+
+namespace LowercaseUrlsWebSite
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            var configuration = app.GetTestConfiguration();
+
+            app.UseServices(services =>
+            {
+                services.AddMvc(configuration);
+
+                services.Configure<RouteOptions>(routeOptions => routeOptions.LowercaseUrls = true);
+            });
+
+            app.UseErrorReporter();
```

Comment:
Please don't use this unless you have a specific need for it. it really degrades the troubleshooting experience when something goes wrong